addOns/ascanrules/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 - Correct Context check in SQL Injection scan rule.
 - "Source Code Disclosure - /WEB-INF folder" is no longer skipped on Java 9+ (Issue 4038).
+- isStop() not checked on IOException leads to max rule duration not enforced (Issue 6647).
```

Comment:
I'd rephrase it to not refer to the Java method, user does not necessarily know about it.

Reply:
Updated